tests/cupy_tests/core_tests/test_fusion.py

```diff
@@ -534,13 +534,21 @@ def sample_function(x, y, z):
         return cupy.square(cupy.add(x, y))
 
     def random_bool(self):
-        return numpy.random.randint(0, 1, (3, 3)) == 0
```

Comment:
What do you mean by split?
These edits are just rewrite of existing test cases with deterministic random arrays.